Composer/plugins/runtimes/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "plugin-runtimes",
+  "version": "1.0.0",
+  "description": "provide info about available runtimes",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "tsc"
+  },
+  "extendsComposer": true,
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "adm-zip": "^0.4.14",
```

Comment:
why are `adm-zip` and `archiver` both required dependencies?

Reply:
remove the unused package.